cmd/checksum_cmd.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"talisman/checksumcalculator"
+	"talisman/gitrepo"
+	"talisman/utility"
+)
+
+type ChecksumCmd struct {
+	fileNamePatterns []string
+}
+
+func NewChecksumCmd(fileNamePatterns []string) *ChecksumCmd {
+	return &ChecksumCmd{fileNamePatterns: fileNamePatterns}
+}
+
+func (s *ChecksumCmd) Run() int {
+	exitStatus := 1
+	wd, _ := os.Getwd()
+	repo := gitrepo.RepoLocatedAt(wd)
+	gitTrackedFilesAsAdditions := repo.TrackedFilesAsAdditions()
+	gitTrackedFilesAsAdditions = append(gitTrackedFilesAsAdditions, repo.StagedAdditions()...)
+	cc := checksumcalculator.NewChecksumCalculator(utility.DefaultSHA256Hasher{}, gitTrackedFilesAsAdditions)
+	rcSuggestion := cc.SuggestTalismanRC(s.fileNamePatterns)
+	if rcSuggestion != "" {
+		fmt.Print(rcSuggestion)
+		exitStatus = 0
```

Comment:
Instead of temp variable at line20, `return 0` at line 29 and `return 1` at line 31

Reply:
I was trying to avoid multiple return points in a function. But if you insist, i shall make this change.

Reply:
I prefer to have less mutation in functions. Temp variable means mutation.

Reply:
Okay, I shall make the code more functional